docs/release-notes.md

```diff
@@ -4,17 +4,28 @@ nav_order: 9
 
 # Release Notes
 
-## Upcoming Ignition 2.17.0 (unreleased)
+## Upcoming Ignition 2.17.1 (unreleased)
 
 ### Breaking changes
 
+### Features
+
+### Changes
+
+### Bug fixes
+
```

Comment:
We need another line between `bug fixes` and the next version to keep the same format we had before.